samples/matter/manufacturer_specific/README.rst

```diff
@@ -0,0 +1,276 @@
+.. _matter_manufacturer_specific_sample:
+
+Matter: Manufacturer-specific
+#############################
+
+.. contents::
+   :local:
+   :depth: 2
+
+This sample demonstrates an implementation of custom manufacturer-specific clusters used by application layer.
+This sample utilizes uses development kit's buttons and LEDs to demonstrate the functionality of the custom ``NordicDevkit`` cluster.
```

Comment:
```suggestion
This sample utilizes development kit's buttons and LEDs to demonstrate the functionality of the custom ``NordicDevkit`` cluster.
```

Reply:
Not a big fan of the word utilize, preferring use.

Reply:
will use "use" then :)